assets/scss/base/_lists.scss

```diff
@@ -1,10 +1,21 @@
+/*
+Lists
 
-/**
-* @desc         lists.
-* @name         layouts
-* @author       Anthony Munene
-* @tested       N/A
-* @requires     N/A
+Unordered lists are used for representing data in a list format.
+
+Markup:
+<ul class="{{modifier_class}}">
+  <li>List item 1</li>
+  <li>List item 2</li>
+  <li>List item 3</li>
+</ul>
+
+.bullets                      - A bulleted list using the modifier `.bullets`
+.lower-letters                - A lowercase ASCII lettered list (a, b, c, etc) using the modifier `.lower-letters`
+.list--ticks                  - A ticked list item with custom tick icons using the modifier `.list--ticks`
+.list--inline                 - An inline list using the modifier `.list--inline`
+
+Styleguide Lists
 */
 
 ul, nav ul {
```

Comment:
@timsb yes this has been done and is now in `base/lists.scss` https://github.com/hmrc/assets-frontend/blob/master/assets/scss/base/_lists.scss#L18-L23